definitions/infra-awsecscluster/summary_metrics.yml

```diff
@@ -1,13 +1,3 @@
-providerAccountName:
```

Comment:
Why are we removing this one? Most of the AWS summaries have them

Reply:
Good catch! Fixed!